backend/app.py

```diff
@@ -21,10 +22,18 @@ def create_user(user: User, session: SessionDep):
     session.add(user)
     session.commit()
     session.refresh(user)
-    
-
     return user
 
+
+@app.post("/weekly")
+def create_weekly(start_date: date, end_date: date, session: SessionDep):
+    weekly = Weekly(start_date=start_date, end_date=end_date)
+    session.add(weekly)
+    session.commit()
+    session.refresh(weekly)
+    return weekly
+
+
```

Comment:
When submit date from frontend to backend , 
fastapi raise the bellow error

INFO:     127.0.0.1:37848 - "POST /weekly HTTP/1.1" 422 Unprocessable Entity

Reply:
Try to set up the debug configuration in the `launch.json` so that you can debug frontend and backend at the same time ;)